src/test_runner.js

```diff
@@ -285,7 +285,7 @@ TestRunner.prototype = {
     var sentry;
 
     var seleniumSessionId;
-    var stdout = "";
+    var stdout = clc.greenBright(logStamp()) + " Magellan child process start\n\n";
```

Comment:
mainly I'm keeping it consistent with what we're already doing. I don't think a finer resolution is going to give us much useful information in this particular use case. The timestamps are mostly here to be compared to one another rather than to a master clock, especially since Magellan's internal scheduler is free to run and re-run tests whenever it wants